examples/serving-tools/modal/run-arcade-worker.py

```diff
@@ -5,9 +5,9 @@
 # Define the FastAPI app
 app = App("arcade-worker")
 
-toolkits = ["arcade-google", "arcade-slack"]
+toolkits = ["arcade_google", "arcade_slack"]
 
-image = Image.debian_slim().pip_install("arcade-ai").pip_install(toolkits)
+image = Image.debian_slim().pip_install("arcade-ai[fastapi]").pip_install(toolkits)
```

Comment:
shouldn't be needed for arcade-ai >=1.0.2. Were you facing issues with this?

Reply:
modal had a weird bug here at one point where they needed you to install FastAPI again after the initial spinup. not sure if this was related or not.

Reply:
Will check it out